pages/profile-page.js

```diff
@@ -72,11 +75,18 @@ export default function ProfilePage({ theme, customTheme, handleToolTip }) {
   const userName = session.user.name;
 
   function getInitials(string) {
```

Comment:
The function looks the same as the one you already have in ProfileLink.js. You could save it in your utils folder and import it from there.